### PR TITLE
2 create note deletion page UI

### LINKED ISSUE #2: Create note deletion page UI

### PR DESCRIPTION
**Pull request by abhishekt98**
Fixes #2

### CODE REVIEW

File: app/javascript/src/components/notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import { Toastr, Alert } from "neetoui/v2";
+
+const Deletealert = ({ isOpen, onClose }) => {
+  const handleDelete = () => {
+    Toastr.success("Note has been successfully Deleted.");
+    onClose();
+  };
+  return (
+    <div className="p-4">
```
Comment (karthiknmenon) [inline app/javascript/src/components/notes/DeleteAlert.jsx:11]: Why do we need a `div` here? Also the classname?

File: app/javascript/src/components/notes/index.jsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
-import React from "react";
+import React, { useState } from "react";
 
 import Container from "components/Container";
 
+import Deletealert from "./DeleteAlert";
 import Heading from "./Heading";
-import Note from "./Note";
+import Note from "./note";
```
Comment (karthiknmenon) [inline app/javascript/src/components/notes/index.jsx:7]: import Note from "./Note";

File: app/javascript/src/components/notes/index.jsx [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
 import data from "../../data";
 import MenuBar from "../menubar";
 
 const Index = () => {
   const notes = data.notes;
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/notes/index.jsx:14]: const [isDeleteAlertOpen, setIsDeleteAlertOpen] = useState(false);

File: app/javascript/src/components/sidebar/index.jsx [hunk 4]
```diff
@@ -26,17 +26,20 @@ const Index = () => {
...
   };
+
+  const dropdownProps = [
+    {
+      label: "Edit",
+    },
+    {
+      label: "Logout",
+    },
+  ];
```
Comment (karthiknmenon) [inline app/javascript/src/components/sidebar/index.jsx:37]: Please move to a `constants` file.

**Review by karthiknmenon — COMMENTED** [review]
@abhishekt98 Please check the suggestions. Also, the folder should be `Notes` instead of `notes`.

File: app/javascript/src/components/notes/DeleteAlert.jsx (hunk 1 above)
Comment (abhishekt98) [reply app/javascript/src/components/notes/DeleteAlert.jsx:11]: yea, that was a mistake. We don't need any div or className there.

**Pull request merged**